Common/include/option_structure.hpp

```diff
@@ -1176,6 +1176,119 @@ static const MapType<std::string, TURB_TRANS_MODEL> Trans_Model_Map = {
   MakePair("LM", TURB_TRANS_MODEL::LM)
 };
 
+/*!
+ * \brief SA Options
+ */
+enum class LM_OPTIONS {
+  NONE,     /*!< \brief No option / default. */
+  LM2015,      /*!< \brief Cross-flow corrections. */
+  MALAN,    /*!< \brief Kind of transition correlation model (Malan). */
+  SULUKSNA,    /*!< \brief Kind of transition correlation model (Suluksna). */
+  KRAUSE,    /*!< \brief Kind of transition correlation model (Krause). */
+  KRAUSE_HYPER,    /*!< \brief Kind of transition correlation model (Krause hypersonic). */
+  MEDIDA_BAEDER,    /*!< \brief Kind of transition correlation model (Medida-Baeder). */
+  MEDIDA,    /*!< \brief Kind of transition correlation model (Medida). */
+  MENTER_LANGTRY,    /*!< \brief Kind of transition correlation model (Menter-Langtry). */
+  DEFAULT    /*!< \brief Kind of transition correlation model (Menter-Langtry if SST, MALAN if SA). */
+};
+
+static const MapType<std::string, LM_OPTIONS> LM_Options_Map = {
+  MakePair("NONE", LM_OPTIONS::NONE)
+  MakePair("LM2015", LM_OPTIONS::LM2015)
+  MakePair("MALAN", LM_OPTIONS::MALAN)
+  MakePair("SULUKSNA", LM_OPTIONS::SULUKSNA)
+  MakePair("KRAUSE", LM_OPTIONS::KRAUSE)
+  MakePair("KRAUSE_HYPER", LM_OPTIONS::KRAUSE_HYPER)
+  MakePair("MEDIDA_BAEDER", LM_OPTIONS::MEDIDA_BAEDER)
+  MakePair("MENTER_LANGTRY", LM_OPTIONS::MENTER_LANGTRY)
+  MakePair("DEFAULT", LM_OPTIONS::DEFAULT)
+};
```

Comment:
Sorry I was forgetting something important. Please add documentation for the LM_OPTIONS to config_template.cfg and to the users guide (https://su2code.github.io/docs_v7/Physical-Definition/#turbulence-models) the website is updated via PR to this repo (https://github.com/su2code/su2code.github.io/blob/master/_docs_v7/Physical-Definition.md) (also pointing to the file you need to change)

Reply:
@rois1995 do you think you will finish this PR in the next week or so? I want to cut a new release and this fits nicely in the "return of transition models to SU2" theme. Otherwise, I'll go ahead with the release early next week (because it has been a while).
If I can help with anything let me know.

Reply:
For sure I will finish it this week. I am starting to work on it now.